src/Tizen.Multimedia.MediaPlayer/Player/Player.Properties.cs

```diff
@@ -444,5 +444,24 @@ public bool IsAudioOnly
                     ThrowIfFailed(this, "Failed to set the audio-only state of the player");
             }
         }
+
+        private SphericalVideo _sphericalVideo;
+
+        /// <summary>
+        /// Gets the 360 video settings.
```

Comment:
How about use a word consistently? 360 or spherical

Reply:
ok i'll change all to spherical